src/test/Readme.txt

```diff
@@ -1,2 +1,19 @@
-check javassist.JvstTestRoot.PATH and .JAR_PATH and then
-run javassist.JvstTest under ./runtest
+# How to run tests
+
+Requirements:
+Java JDK 9
+Maven
+
+1) Build jar file and move it to the top level folder.
+
+    > mvn package
+    > mv ./target/javassist*-GA.jar ./javaassist.jar
+
+2) Check that ./src/test/javassist/JvstTestRoot.PATH and .JAR_PATH point to the compiled jar file.
+   The default is "../".
+
+3) Run Tests
+
+    > mvn test
+    > mvn surefire:test
+
```

Comment:
Oh was there a Readme for tests... I don't like this!

You shouldn't have to look for documentation to be able to run tests they should simply work by running:

mvn test